cmd/meroxa/root/apps/deploy.go

```diff
@@ -601,21 +616,21 @@ func (d *Deploy) checkResourceAvailability(ctx context.Context) error {
 		return errors.New("no resources defined in your Turbine app")
 	}
 
-	if errStr := d.getResourceCheckErrorMessage(ctx, resources); errStr != "" {
-		errStr += ";\n\n ⚠️  Run 'meroxa resources list' to verify that the resource names " +
-			"defined in your Turbine app are identical to the resources you have created on the Meroxa Platform before deploying again"
+	if err := d.validateResources(ctx, resources); err != nil {
 		d.logger.StopSpinnerWithStatus("Resource availability check failed", log.Failed)
-		return fmt.Errorf("%s", errStr)
+		return fmt.Errorf("%w;\n\n%s", err, resourceInvalidError)
 	}
 
-	if !d.flags.SkipCollectionValidation {
-		if err := d.validateCollections(ctx, resources); err != nil {
-			d.logger.StopSpinnerWithStatus("Resource availability check failed", log.Failed)
-			return err
-		}
+	if d.flags.SkipCollectionValidation {
+		d.logger.StopSpinnerWithStatus("Can access your Turbine resources", log.Successful)
+		return nil
+	}
+
+	if err := d.validateCollections(ctx, resources); err != nil {
+		d.logger.StopSpinnerWithStatus("Resource availability check failed", log.Failed)
+		return err
 	}
 
-	d.logger.StopSpinnerWithStatus("Can access your Turbine resources", log.Successful)
```

Comment:
You'll need to return this log line. Otherwise the cli goes crazy

Reply:
uh oh you are right!  thanks

Reply:
fixed in 798a4c17